src/components/Preferences/Availability.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+    Box,
+    Checkbox,
+    HStack,
+    Flex,
+    Select,
+    Stack,
+    Text,
+} from "@chakra-ui/react";
+import { useState } from "react";
+
+const Availability = () => {
+    const [isReliefDisabled, setIsReliefDisabled] = useState(true);
+    const [isSabbaticalDisabled, setIsSabbaticalDisabled] = useState(true);
+
+    return (
+        <Box>
+            <Stack spacing={5} direction="column">
+                <HStack spacing={5}>
+                    <Text>Please check desired semester(s) off:</Text>
+                    <Checkbox> Fall </Checkbox>
+                    <Checkbox> Spring </Checkbox>
+                    <Checkbox> Summer </Checkbox>
+                </HStack>
+                <Stack spacing={1} direction="column">
+                    <Checkbox
+                        onChange={() => setIsReliefDisabled(!isReliefDisabled)}
+                    >
+                        {" "}
+                        I am taking relief{" "}
+                    </Checkbox>
+                    <HStack>
+                        <Text minW="450px">
+                            If yes, enter how many courses you will be teaching
+                            this year:
+                        </Text>
+                        <Select
+                            placeholder="5 Courses"
+                            isDisabled={isReliefDisabled}
+                        >
+                            <option value="option2">4 Courses</option>
+                            <option value="option3">3 Courses</option>
+                            <option value="option4">2 Courses</option>
+                            <option value="option5">1 Course</option>
```

Comment:
If this issue does not include anything to do with forms, this may be fine, but ideally the values should line up with the options, maybe "1" "2" "3" etc..

Reply:
The order they appear in the dropdown makes sense to be descending, would it be weird to have the options also be descending so they match?